pycodestyle.py

```diff
@@ -275,6 +276,9 @@ def maximum_line_length(physical_line, max_line_length, multiline, noqa):
     line = physical_line.rstrip()
     length = len(line)
     if length > max_line_length and not noqa:
+        # Special case: ignore long shebang lines.
+        if line_number == 1 and length > 1 and line[:2] == '#!':
```

Comment:
Is `length > 1` required here? We're in the base where `length > max_line_length` and the slice you're using works even on empty strings:

```py
>>> ''[:2]
''
```

And `''` can be safely compared to `#!`

Reply:
Haha @sigmavirus24 [Jynx](https://github.com/PyCQA/pycodestyle/pull/736#discussion_r178100679) ;)

Reply:
12 minutes versus 8 minutes 😛

Reply:
![](http://i0.kym-cdn.com/photos/images/newsfeed/000/181/367/closeenough.png?1317606898)